tools/build_browser.js

```diff
@@ -75,11 +75,39 @@ async function installDemo(languages, { minify }) {
     && lang.sample
   );
 
-  const css = await glob("styles/*.css", {cwd:"./src"})
-  const styles = css.map((el) => (
-    { "name": _.startCase(path.basename(el,".css")), "path": el }
-  ));
-  renderTemplate("./demo/index.html", "./demo/index.html", { styles, languages, minify });
+  languages.forEach((language) => {
```

Comment:
Can all this be pulled out into a `renderIndex()` function?  installDemo is getting a bit long.

Reply:
Done.